build/build-buildimages.sh

```diff
@@ -81,7 +81,13 @@ touch $ACR_BUILD_IMAGES_ARTIFACTS_FILE
 > $ACR_BUILD_IMAGES_ARTIFACTS_FILE
 
 # Build buildpack images
-source $REPO_DIR/build/build-buildpacks-images.sh
+# 'pack' is not supported on Windows
```

Comment:
this is incorrect

Reply:
You mean should it be `package create-builder`?